test/OrchardCore.Commerce.Tests.UI/Helpers/SetupHelpers.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Pages;
+using Lombiq.Tests.UI.Services;
+using OpenQA.Selenium;
+
+namespace OrchardCore.Commerce.Tests.UI.Helpers;
+
+public static class SetupHelpers
+{
+    public const string RecipeId = "Blog";
```

Comment:
Please add a setup recipe that calls Blog as well as the Product and Order recipes and use it here. This is to prove the OCC features don't interfere with the basic functionality.

Reply:
Btw I wanted to use Sofware as a Service but for some reason it produced an error while the Blog is working fine.

Reply:
The recipe ID here remains the same so the recipe you created isn't actually used.

Reply:
Adding the recipe to the recipeId breaks the tests. This is my first time setting up this whole UI test system and recipes.
Could you please help me spot the problem? I observed FailureDumps and it doesn't get past the setup site stage.
![3](https://user-images.githubusercontent.com/92299130/171058566-7847ded4-a198-47d2-8e1a-32126a988e8f.png)
Also it couldn't set the recipe to blog.

Reply:
The is pushed also